### DISCUSSION TITLE
Verification / Reverification with Freshdesk

### DISCUSSION THREAD

**Discussion by jitendraag** [Coronasafe/life#202] [Ideas]
Our current infrastructure and methodology allows for distributed verification of covid resources. We want to augment this verification with volunteer capacity that has been offered to CRIA directly.

In order to save time we will start verification with a Glide App - https://covidvolunteers.glideapp.io/, however we want to utilise Freshdesk's ticketing and automation capabilities.

Here are some things Freshdesk can do for us:
1. Enable unlimited volunteers to join and help with verification
2. Ensure same resource is not being verified by other volunteers
3. Ensure resources verified 30 minutes ago can become valid for re-verification

We will have to integrate with Freshdesk APIs: https://developers.freshdesk.com/api/

1. Add a column (freshdesk ticket id) to database
**2. A cron job (or similar) that looks at all entries in master database for empty Freshdesk ticket id**
3. All 'new resources' (where ticket id is empty) use Create API to create a ticket and save ticket id in master database
4. Whenever volunteers verify a resource, they change status on Freshdesk. Freshdesk can call a webhook and this webhook can update the entry in master database.

This completes the two way loop for new leads and verification / re-verification.

We want to discuss this and consider it for a new feature request. We will arrange for Freshdesk license, keys etc.

**Comment by karmakoder** [comment]
@jitendraag i havent used freshdesk before but hope ramping volunteers on it will be smooth/quick. Glide app looks pretty strightfwd but my hunch is freshdesk can give a higher level pulse-check and volunteer management layer, which makes sense. New master db is postgres so hopefully the webhook from freshdesk can come directly to the CRUD layer of postgres. I'll double check about postgres once folks are back online. Overall, makes sense to use some fleet management solution. FreshDesk appears to be one.

**Comment by vasanthirajkumar** [comment]
Kindly update the location to Mumbai for Oxygen supplier list at chennai district.
He is actually in Mumbai. Thanks much

Bharat Bhai Michael (Area Manager At Philips)
Chennai
Quantity Available:
0
Price:
₹ 0
9867031020
